adoc/release-2.0.adoc

```diff
@@ -0,0 +1,115 @@
+// Start attribute entry list (Do not edit here! Edit in entities.adoc)
+ifdef::env-github[]
+:suse: SUSE
+:current-year: 2020
+:product: {suse} Cloud Application Platform
+:version: 2.0
+:rn-url: https://www.suse.com/releasenotes
+:doc-url: https://documentation.suse.com/suse-cap/2
+:deployment-url: https://documentation.suse.com/suse-cap/2.0/single-html/cap-guides/#part-cap-deployment
+:caasp: {suse} Containers as a Service Platform
+:caaspa: {suse} CaaS Platform
+:ostack: OpenStack
+:cf: Cloud Foundry
+:kubecf: KubeCF
+:k8s: Kubernetes
+:scc: {suse} Customer Center
+:azure: Microsoft Azure
+:aks: Azure {k8s} Service
+:aksa: AKS
+:aws: Amazon Web Services
+:awsa: AWS
+:eks: Amazon Elastic Container Service for Kubernetes
+:eksa: Amazon EKS
+:gke: Google Kubernetes Engine
+:gkea: GKE
+:mysql: MySQL
+:mariadb: MariaDB
+:postgre: PostgreSQL
+:redis: Redis
+:mongo: MongoDB
+:ng: NGINX
+endif::[]
+// End attribute entry list
+
+[id='sec.major-change']
+== Major Changes
+
+[id='sec.2_0']
+=== Release 2.0, June 2020
+
+[id='sec.2_0.new']
+==== What Is New?
+* {kubecf} 2.2.2 replaces {scf}:
+** cf-deployment has been updated to TODO-VERSION
+** TODO
+* {operator} TODO-VERSION has been added:
+** TODO
+* Stratos Console has been updated to version 3.2.0:
+** TODO
+** For a full list of features and fixes see https://github.com/SUSE/stratos/blob/master/CHANGELOG.md#320.
+* Stratos Metrics has been updated to version 1.2.1:
+** TODO
+** For a full list of features and fixes see https://github.com/SUSE/stratos-metrics/blob/master/CHANGELOG.md#121.
+
+[id='sec.2_0.feature']
+==== Features and Fixes
+* TODO
+* Includes these {cf} component versions:
+** app-autoscaler: 3.0.0
+** bits-services: 2.28.0
+** bpm: 1.1.6
+** capi: 1.89.0
+** cf-acceptance-tests: 12.17
+** cf-deployment: 12.17
```

Comment:
Upstream is our base reference: https://github.com/cloudfoundry/cf-deployment/blob/v12.36.0/cf-deployment.yml#L2344-L2468 so that's the 1st step. Helm chart values then overrides some of those.